javascript/lang/best-practice/lazy-load-module.yaml

```diff
@@ -0,0 +1,16 @@
+rules:
+- id: lazy-load-module
+  patterns:
+  - pattern: require(...)
+  - pattern-inside: |
+      function $NAME(...) {
+          ...
+      }
+  message: |
+    Lazy loading can complicate code bundling if care is not taken.
```

Comment:
What's the recommended fix for this? Just to do global requiring?

Reply:
Yes! and I definitely need to add this information to the message

Reply:
done!